multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -2824,6 +2829,86 @@ def load(
 
         return predictor
 
+    def dump_timm_image(
+        self,
+        path: str,
+    ):
+        """
+        Save TIMM image model weights and config to local directory.
+        Model weights are saved in file `pytorch_model.bin`;
+        Configs are saved in file `config.json`
+
+        Parameters
+        ----------
+        path : str
+            Path to directory where models and configs should be saved.
+        """
+        models = []
+        if isinstance(self._model, (MultimodalFusionMLP, MultimodalFusionTransformer)) and isinstance(
```

Comment:
@zhiqiangdon We can later add a `BaseMultimodalFusionModel` class, and ensure that `MultimodalFusionMLP` and `MultimodalFusionTransformer` inherit from this class.

Reply:
@suzhoum May be we can add a TODO item here

Reply:
Sure. We can add a base class.